frontend/src/pages/instance/components/InstanceForm.vue

```diff
@@ -347,6 +345,10 @@ export default {
         applicationFieldsVisible: {
             default: false,
             type: Boolean
+        },
+        preDefinedInputs: {
+            default: null,
+            type: Object
```

Comment:
Worth just adding a comment here to explain this allows pre-selection/definition of Instance options in the form - I know the name is well-defined, but nice to be explicit